instrumentation/http/test/instrumentation/http/patches/client_test.rb

```diff
@@ -19,7 +19,7 @@
     @orig_propagation = OpenTelemetry.propagation
     propagator = OpenTelemetry::Trace::Propagation::TraceContext.text_map_propagator
     OpenTelemetry.propagation = propagator
-    instrumentation.install({})
+    instrumentation.install(hide_query_params: true)
```

Comment:
```suggestion
    instrumentation.install({})
```

Reply:
I don't think this should get changed as we want to prevent a regression from the default behavior changing

Reply:
I believe it has to be like this because it's set as false in another test and as a singleton the state is shared across tests so it could be set to false prior to this. I've run into a similar issue with this before and tests were flakey depending on the order they were run. The `Instrumentation::Base` class needs some enhancements to make it more test friendly, but I'm reluctant to do it in the scope of this PR.

Reply:
i see, fair enough.